docs/tutorial/node-app.rst

```diff
@@ -116,9 +116,22 @@ service:
     2023-10-30T12:37:33.663Z [pebble] Service "app" starting: node server.js
     2023-10-30T12:37:33.864Z [app] Running on http://0.0.0.0:8080
 
-Next, open your web browser and navigate to ``http://localhost:8000``. You
-should see a blank page with a "Hello World from inside the rock!" message.
-Success!
+Next we'll verify that the Node.js app is up and running. If you're running this
+tutorial on a regular Ubuntu system, open your web browser and navigate to
+``http://localhost:8000``. You should see a blank page with a
+"Hello World from inside the rock!" message. Success!
+
+If, instead, you are following the tutorial through a Multipass VM, you can
+open another shell into the VM and access the app using curl:
```

Comment:
```suggestion
If, instead, you're working in a Multipass VM, you can
open another shell into the VM and access the app with curl:
```

Simple contractions with pronouns improve readability ("you are" → "you're").

I try and cut down on the instances of "use" and "using" to avoid repetition and make the phrase's core verb (`access`) more prominent, but YMMV.

Reply:
makes sense!